lib/tls.js

```diff
@@ -3,6 +3,7 @@
 const net = require('net');
 const url = require('url');
 const util = require('util');
+const binding = process.binding('crypto');
```

Comment:
I think the different naming is possibly to differentiate it from the result of `require('crypto')`.